filechanges/opt/musicbox/setsound.sh

```diff
@@ -179,33 +179,56 @@ then
 cat << EOF > /etc/asound.conf
 pcm.!default {
     type plug
+    slave.pcm plugequal;
+}
+pcm.plugequal {
+    type dmix
+    ipc_key 1024
+    controls "/home/mopidy/.alsaequal.bin"
```

Comment:
Do you think ```/var/lib/alsa/alsaequal.bin``` would be appropriate? It is not a folder under Musicbox's control, but seems like a good candidate location for storing alsa-related configuration?